test.js

```diff
@@ -402,3 +402,17 @@ test('is.empty', t => {
 	tempSet.add(1);
 	t.false(m.empty(tempSet));
 });
+
+test('is.any', t => {
```

Comment:
Missing test cases:
1. no items (aka `m.any(m.integer)`)
1. invalid predicate? what if the passed predicate is not a function? this should be validated

@sindresorhus say `is.function(predicate) === false`, do we throw an Error?

Reply:
> say is.function(predicate) === false, do we throw an Error?

Yes